src/config/content/es.js

```diff
@@ -0,0 +1,5 @@
+export default {
+	sentenceLength: {
+		recommendedWordCount: 25,
```

Comment:
It would be really nice if you would document the reasoning behind this change here (basically, the text of the issue). So that we have less magical numbers in the code.